poppy/tests/test_fresnel.py

```diff
@@ -789,44 +793,50 @@ def test_CompoundOpticalSystem_hybrid(npix=128):
                                err_msg="PSFs do not match between equivalent simple and compound/hybrid optical systems")
 
 
-def test_inwave_fresnel(plot=False):
-    '''Verify basic functionality of the inwave kwarg for a basic FresnelOpticalSystem()'''
-    npix = 128
-    oversample = 2
+def test_FixedSamplingImagePlaneElement(display=False):
+    poppy_tests_fpath = os.path.abspath(poppy.__file__)[:-11]+'tests/'
```

Comment:
The `[:-11]` is a bit of a kludge for accessing a directory name here. Better would be something using the os.path functions, for instance `os.path.dirname(os.path.abspath(__file__))`

[optional]